src/state/update-user.js

```diff
@@ -0,0 +1,21 @@
+export const updateUser = (uname, edit) => {
+  console.log("there");
+  console.log("uname", uname);
+  fetch(`https://m07.herokuapp.com/${uname}`, {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json"
+    },
+    mode: "cors",
+    body: JSON.stringify(edit)
+  })
+    .then(response => response.json())
+    .then(() => {
+      console.log("now");
```

Comment:
Next steps? You are fetching the json, do we then want to reset uname values?

Reply:
This is a function that posts the updated user using the uname prop which is pulled through from the editview component. So in short, no.

Reply:
We could reroute to the profile view instead, but I think that would be done in a different component.

Reply:
It looks good